system/indy-node-tests/test_freshness.py

```diff
@@ -1,7 +1,6 @@
 import pytest
 import time
 import asyncio
-from indy import did, payment
```

Comment:
Do we still need to import did?

Reply:
I updated the pr, and used Ruff to remove unused imports. No, we do not need to import "did" anymore.

Reply:
I see why you ask, It was not needed before this pr.